src/components/InfoBanner/docs/InfoBanner.mdx

```diff
@@ -0,0 +1,27 @@
+---
+name: InfoBanner
+menu: Components
+route: /components/info-banner
+---
+
+import { Playground } from 'docz';
+import { InfoBanner } from '../InfoBanner'
+import { InfoBannerPropsTable } from './InfoBannerPropsTable'
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks'
+
+# InfoBanner
+
+Vibrant information messages used as page banners or within forms.
+
```

Comment:
Do you think we could add a little more description and indication of intended usage to Docs?

Something like:

# Usage
Banner is used to communicate a prominent message, state change or important contextual information that affects the system, feature, running process or the whole page. It also can contain related optional actions.